docs/app/Examples/modules/Dropdown/Content/DropdownExampleHeader.js

```diff
@@ -1,15 +1,16 @@
 import React from 'react'
 import { Dropdown } from 'semantic-ui-react'
 
-const DropdownExampleHeader = () => (
-  <Dropdown text='Filter' icon='filter' floating labeled button className='icon'>
-    <Dropdown.Menu>
-      <Dropdown.Header icon='tags' content='Filter by tag' />
-      <Dropdown.Item>Important</Dropdown.Item>
-      <Dropdown.Item>Announcement</Dropdown.Item>
-      <Dropdown.Item>Discussion</Dropdown.Item>
-    </Dropdown.Menu>
-  </Dropdown>
-)
+const menu = {
+  items: [
+    { component: Dropdown.Header, icon: 'tags', content: 'Filter by tag' },
+    { text: 'Important', value: 'Important' },
+    { text: 'Hopper', value: 'Hopper' },
+    { text: 'Discussion', value: 'Discussion' },
+  ],
+}
+
+const DropdownExampleHeader = () =>
+  <Dropdown text='Filter' icon='filter' floating labeled button className='icon' menu={menu} />
```

Comment:
I like this API the most.  This is the most consistent with where we are headed with shorthand and subcomponent alignment.